src/ast/nodes/NodeType.ts

```diff
@@ -1,64 +1,127 @@
+export const AwaitExpression = 'AwaitExpression';
+export const BinaryExpression = 'BinaryExpression';
+export const BlockStatement = 'BlockStatement';
+export const BreakStatement = 'BreakStatement';
+export const CallExpression = 'CallExpression';
+export const CatchClause = 'CatchClause';
+export const ClassBody = 'ClassBody';
+export const ClassDeclaration = 'ClassDeclaration';
+export const ClassExpression = 'ClassExpression';
+export const ConditionalExpression = 'ConditionalExpression';
+export const DoWhileStatement = 'DoWhileStatement';
+export const EmptyStatement = 'EmptyStatement';
+export const ExportAllDeclaration = 'ExportAllDeclaration';
+export const ExportDefaultDeclaration = 'ExportDefaultDeclaration';
+export const ExportNamedDeclaration = 'ExportNamedDeclaration';
+export const ExportSpecifier = 'ExportSpecifier';
+export const ExpressionStatement = 'ExpressionStatement';
+export const ForStatement = 'ForStatement';
+export const ForInStatement = 'ForInStatement';
+export const ForOfStatement = 'ForOfStatement';
+export const FunctionDeclaration = 'FunctionDeclaration';
+export const FunctionExpression = 'FunctionExpression';
+export const Identifier = 'Identifier';
+export const IfStatement = 'IfStatement';
+export const Import = 'Import';
+export const ImportDeclaration = 'ImportDeclaration';
+export const ImportDefaultSpecifier = 'ImportDefaultSpecifier';
+export const ImportNamespaceSpecifier = 'ImportNamespaceSpecifier';
+export const ImportSpecifier = 'ImportSpecifier';
+export const LabeledStatement = 'LabeledStatement';
+export const Literal = 'Literal';
+export const LogicalExpression = 'LogicalExpression';
+export const MemberExpression = 'MemberExpression';
+export const MetaProperty = 'MetaProperty';
+export const MethodDefinition = 'MethodDefinition';
+export const NewExpression = 'NewExpression';
+export const ObjectExpression = 'ObjectExpression';
+export const ObjectPattern = 'ObjectPattern';
+export const Program = 'Program';
+export const Property = 'Property';
+export const RestElement = 'RestElement';
+export const ReturnStatement = 'ReturnStatement';
+export const SequenceExpression = 'SequenceExpression';
+export const SpreadElement = 'SpreadElement';
+export const SwitchCase = 'SwitchCase';
+export const SwitchStatement = 'SwitchStatement';
+export const TaggedTemplateExpression = 'TaggedTemplateExpression';
+export const TemplateElement = 'TemplateElement';
+export const TemplateLiteral = 'TemplateLiteral';
+export const ThisExpression = 'ThisExpression';
+export const ThrowStatement = 'ThrowStatement';
+export const TryStatement = 'TryStatement';
+export const UnaryExpression = 'UnaryExpression';
+export const UpdateExpression = 'UpdateExpression';
+export const VariableDeclarator = 'VariableDeclarator';
+export const VariableDeclaration = 'VariableDeclaration';
+export const WhileStatement = 'WhileStatement';
+export const YieldExpression = 'YieldExpression';
```

Comment:
The only thing I do not like here is that there is no guarantee there is no typo between the types and the constants. Maybe each constant could also be explicitly given its type?